Yoshi/Yoshi/YoshiTableViewSelectable.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  YoshiTableViewSelectable.swift
+//  Yoshi
+//
+//  Created by Daniel Vancura on 4/9/16.
+//  Copyright © 2016 Prolific Interactive. All rights reserved.
+//
+
+/** 
+ A selectable item inside a debugging table view.
+ */
+public protocol YoshiTableViewSelectable: class {
+
+    // Indicates whether or not this item is selectable.
+    var selected: Bool { get set }
```

Comment:
Yeah, this might make more sense as long as the selected state is something we require for every `YoshiTableViewMenuItem` anyways - as far as I see it, there are no other objects that might benefit from a `selected` state, so: :+1: